### PR TITLE
Add .ruby-version to match travis version

### PR DESCRIPTION
**Pull request by Firefishy**
Add a .ruby-version to set local ruby version for rvm and rbenv users.

### CODE REVIEW

**Comment by stephengroat** [comment]
@Firefishy thanks for the addition, great idea!

where did you get that `github` is running `2.4.5`? do you have a link to the source of that info? from https://github.com/github/pages-gem, it seems like its actually `2.5.3`

**Comment by Firefishy** [comment]
@stephengroat Updated ruby to `2.5.3` now to match version across all 3: Local dev, TravisCI and github.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 sudo: false
-rvm: 2.4.0
+rvm: 2.5.3
```
Comment (stephengroat) [inline .travis.yml]:
```suggestion
```
have travis use `.ruby-version`
https://docs.travis-ci.com/user/languages/ruby/#using-ruby-version

**Review by stephengroat — APPROVED** [review]
👍

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+2.5.3
```
Comment (Carlgo11) [inline .ruby-version:1]: What are the benefits of having the ruby version in `.ruby-version` as opposed to .travis.yml and Gemfile?

Comment (stephengroat) [reply .ruby-version:1]: travis doesn't recognize `Gemfile` ruby version, `rvm` and travis will use `.ruby_version`

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**